pages/_document.js

```diff
@@ -2,8 +2,6 @@ import Document, { Html, Head, Main, NextScript } from "next/document";
 import { ServerStyleSheet } from "styled-components";
 import { getSiteString } from "@/lib/utils";
 
-const GOOGLE_APP_ID = process.env.NEXT_PUBLIC_GOOGLE_APP_ID;
```

Comment:
Not sure if this is suppose to be here or not....but sure doesn't seem to be in use in this file, so I removed